cellprofiler/modules/smooth.py

```diff
@@ -40,90 +40,102 @@ class Smooth(cpm.Module):
-            <ul>
-            <li><i>%(FIT_POLYNOMIAL)s:</i> This method treats the intensity of the image pixels
-            as a polynomial function of the x and y position of
-            each pixel. It fits the intensity to the polynomial,
-            <i>A x<sup>2</sup> + B y<sup>2</sup> + C xy + D x + E y + F</i>.
-            This will produce a smoothed image with a single peak or trough of intensity
-            that tapers off elsewhere in the image. For many microscopy images (where
-            the illumination of the lamp is brightest in the center of field of view),
-            this method will produce an image with a bright central region and dimmer
-            edges. But, in some cases the peak/trough of the polynomial may actually
-            occur outside of the image itself.</li>
-            <li><i>%(GAUSSIAN_FILTER)s:</i> This method convolves the image with a Gaussian whose
-            full width at half maximum is the artifact diameter entered.
-            Its effect is to blur and obscure features
-            smaller than the artifact diameter and spread bright or
-            dim features larger than the artifact diameter.</li>
-            <li><i>%(MEDIAN_FILTER)s:</i> This method finds the median pixel value within the
-            artifact diameter you specify. It removes bright or dim features that are much smaller
-            than the artifact diameter.</li>
-            <li><i>%(SMOOTH_KEEPING_EDGES)s:</i> This method uses a bilateral filter which
-            limits Gaussian smoothing across an edge while
-            applying smoothing perpendicular to an edge. The effect
-            is to respect edges in an image while smoothing other
-            features. <i>%(SMOOTH_KEEPING_EDGES)s</i> will filter an image with reasonable
-            speed for artifact diameters greater than 10 and for
-            intensity differences greater than 0.1. The algorithm
-            will consume more memory and operate more slowly as
-            you lower these numbers.</li>
-            <li><i>%(CIRCULAR_AVERAGE_FILTER)s:</i> This method convolves the image with
-            a uniform circular averaging filter whose size is the artifact diameter entered. This filter is
-            useful for re-creating an out-of-focus blur to an image.</li>
-            <li><i>%(SM_TO_AVERAGE)s:</i> Creates a flat, smooth image where every pixel
-            of the image equals the average value of the original image.</li>
-            </ul>""" % globals())
+                 SM_TO_AVERAGE], doc="""\
+This module smooths images using one of several filters. Fitting a
+polynomial is fastest but does not allow a very tight fit compared to
+the other methods:
+
+-  *%(FIT_POLYNOMIAL)s:* This method treats the intensity of the image
+   pixels as a polynomial function of the x and y position of each
+   pixel. It fits the intensity to the polynomial, *A x :sup:`2` + B
```

Comment:
The superscripts don't seem to be rendering properly with the whole equation in italics;  try:

```
*A x* :sup:`2` *+ B y* :sup:`2` *+ C xy + D x + E y + F*
```